orc8r/cloud/go/services/configurator/storage/sql.go

```diff
@@ -472,6 +472,12 @@ func (store *sqlConfiguratorStorage) LoadEntities(networkID string, filter Entit
 	}
 	sort.Slice(ret.Entities, func(i, j int) bool { return entComparator(ret.Entities[i], ret.Entities[j]) })
 
+	// Ensure there are more pages to return
+	if len(ret.Entities) != 0 && uint64(len(ret.Entities)) == loadCriteria.PageSize {
```

Comment:
```
if len(ret.Entities) == loadCriteria.GetPageSize() {
  ...
}


// and elsewhere

// and note that lc.PageSize of 0 is magic value
GetPageSize() int {
  max := configurator default
  if lc.PageSize == 0 {
    return max
  }
  return min(lc.PageSize, max)
}
```

Reply:
Given that the loadCriteria here is a proto, I had to make this an unexported function instead